smithy-openapi/src/test/java/software/amazon/smithy/openapi/fromsmithy/protocols/AwsRestJson1ProtocolTest.java

```diff
@@ -266,4 +266,27 @@ public OperationObject updateOperation(
             return OpenApiMapper.super.updateOperation(context, shape, operation, httpMethodName, path);
         }
     }
+
+    @Test
+    public void convertsExamples() {
+        Model model = Model.assembler()
+                .addImport(getClass().getResource("examples-test.smithy"))
+                .discoverModels()
+                .assemble()
+                .unwrap();
+        OpenApiConfig config = new OpenApiConfig();
+        config.setService(ShapeId.from("smithy.examplestrait#Banking"));
+        ObjectNode result = OpenApiConverter.create()
+                .config(config)
+                .convertToNode(model);
+        InputStream openApiStream = getClass().getResourceAsStream("examples-test.openapi.json");
+
+        if (openApiStream == null) {
+            fail("OpenAPI model not found for test case: " + "examples-test.openapi.json");
```

Comment:
Change this to throwing a `RuntimeException` with the same message. There is a meaningful distinction between a failed test and a test throwing an exception.

Reply:
Changed to throwing a `RuntimeException` with the same message.